recipe/meta.yaml

```diff
@@ -12,34 +12,47 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  Skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python
+    - wheel
   run:
-    - __{{ target_os }}
-    {% if target_os == 'win' %}
-    - ansicon
-    {% endif %}
-    - python >=3.8
+    - ansicon  # [win]
+    - python
 
 test:
+  source_files:
+    - tests
   imports:
     - jinxed
     - jinxed.terminfo
   commands:
     - pip check
+    - pytest -vv tests  # [py<311]
+    # seems from the tests that jinxed 1.2.0 does not support well compability tests 
+    # with ncurses >= 6.3, ncurses < 6.3 is not available for py>=311
+    - pytest -k "not test_tparm.py" -vv  # [py>=311 or not win]
   requires:
     - pip
+    - pytest
+    - ncurses <6.3 # [py<311 or not win]
```

Comment:
```suggestion
```
All our python are built with `- ncurses >=6.4,<7.0a0`. By setting <6.3 you are restricting to old builds of 3.10, 3.9, 3.8. Skipping the tests with a comment is fine. You can open an issue upstream too.

Reply:
raised issue in upstream https://github.com/Rockhopper-Technologies/jinxed/issues/4